### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by creikey**

### CODE REVIEW

**Comment by ColinLi33** [comment]
LGTM!

**Comment by creikey** [comment]
Made a spelling mistake in the youtube link

**Comment by ColinLi33** [comment]
LGTM!

**Comment by Dellamoresteven** [comment]
What is the first comment? Is that his face?

**Comment by ColinLi33** [comment]
> What is the first comment? Is that his face?

Wow that's interesting...

**Comment by creikey** [comment]
> What is the first comment? Is that his face?

I accidentally dragged and dropped from the github profile, undid it in the second commit

**Comment by Dellamoresteven** [comment]
> > What is the first comment? Is that his face?
> 
> I accidentally dragged and dropped from the github profile, undid it in the second commit

ok?

**Comment by creikey** [comment]
> 

?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # gcc-explorer
 My attempt at making GCC error logs more readable
 
+![image](https://user-images.githubusercontent.com/28424206/159104561-22e774d1-a659-4e7c-9259-7d457e2ab1d5.png)
```
Comment (ColinLi33) [inline README.md]: Why is there a picture of a face? Is this him?

**Review by ColinLi33 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (creikey) [reply README.md]: It was an accident new github ui shows weird on my monitor 🙄

Comment (ColinLi33) [reply README.md]: thats pretty sus ඞඞ

Comment (creikey) [reply README.md]: > 

what?

**Comment by creikey** [comment]
image

**Comment by Dellamoresteven** [comment]
> image

?

**Comment by ColinLi33** [comment]
what is this?

**Comment by creikey** [comment]
image

Why did you send this?

**Comment by ColinLi33** [comment]
@Dellamoresteven what is this guy looking at?

**Pull request closed without merging**